spec/intl.html

```diff
@@ -0,0 +1,584 @@
+          1. <del>Let _bestFormat_ be BasicFormatMatcher(_opt_, _formats_).</del>
+        1. <del>Else,</del>
+          1. <del>Let _bestFormat_ be BestFitFormatMatcher(_opt_, _formats_).</del>
+        1. <del>For each row in <emu-xref href="#table-datetimeformat-components"></emu-xref>, except the header row, in table order, do</del>
+          1. <del>Let _prop_ be the name given in the Property column of the row.</del>
+          1. <del>Let _p_ be _bestFormat_.[[&lt;_prop_&gt;]].</del>
+          1. <del>If _p_ not *undefined*, then</del>
+            1. <del>Set _dateTimeFormat_'s internal slot whose name is the Internal Slot column of the row to _p_.</del>
+        1. <del>If _dateTimeFormat_.[[Hour]] is not *undefined*, then</del>
+          1. <del>Let _hcDefault_ be _dataLocaleData_.[[hourCycle]].</del>
+          1. <del>Let _hc_ be _dateTimeFormat_.[[HourCycle]].</del>
+          1. <del>If _hc_ is *null*, then</del>
+            1. <del>Set _hc_ to _hcDefault_.</del>
+          1. <del>If _hour12_ is not *undefined*, then</del>
+            1. <del>If _hour12_ is *true*, then</del>
+              1. <del>If _hcDefault_ is `"h11"` or `"h23"`, then</del>
+                1. <del>Set _hc_ to `"h11"`.</del>
+              1. <del>Else,</del>
+                1. <del>Set _hc_ to `"h12"`.</del>
+            1. <del>Else,</del>
+              1. <del>Assert: _hour12_ is *false*.</del>
+              1. <del>If _hcDefault_ is `"h11"` or `"h23"`, then</del>
+                1. <del>Set _hc_ to `"h23"`.</del>
+              1. <del>Else,</del>
+                1. <del>Set _hc_ to `"h24"`.</del>
+          1. <del>Set _dateTimeFormat_.[[HourCycle]] to _hc_.</del>
+          1. <del>If _dateTimeformat_.[[HourCycle]] is `"h11"` or `"h12"`, then</del>
+            1. <del>Let _pattern_ be _bestFormat_.[[pattern12]].</del>
+          1. <del>Else,</del>
+            1. <del>Let _pattern_ be _bestFormat_.[[pattern]].</del>
+        1. <del>Else,</del>
+          1. <del>Set _dateTimeFormat_.[[HourCycle]] to *undefined*.</del>
+          1. <del>Let _pattern_ be _bestFormat_.[[pattern]].</del>
+        1. Set _dateTimeFormat_.[[Pattern]] to <del>_pattern_</del><ins>GetDateTimeFormatPattern(_opt_, _formats_, _hc_)</ins>.
+        1. <ins>For each row in <emu-xref href="#table-temporal-patterns"></emu-xref>, except the header row, in table order, do</ins>
+          1. <ins>Let _limitedOptions_ be a new Record.</ins>
+          1. <ins>Let _fields_ be the list of fields in the Fields column of the row.</ins>
+          1. <ins>For each field _field_ of _opts_, do</ins>
+            1. <ins>If _field_ is in _fields_, then</ins>
+              1. <ins>Set _limitedOptions_.[[&lt;_field_>]] to _opts_.[[&lt;_field_>]].</ins>
+          1. <ins>Let _bestFormat_ be GetDateTimeFormatPattern(_limitedOptions_, _formats_, _hc_).</ins>
+          1. <ins>If _bestFormat_ does not have any fields that are in _fields_, then</ins>
```

Comment:
Maybe also do something if _bestFormat_ has any fields that are not in _fields_.  (See my other comment)